actionpack/test/controller/parameters/accessors_test.rb

```diff
@@ -77,11 +77,15 @@ class ParametersAccessorsTest < ActiveSupport::TestCase
 
   test "each_value carries permitted status" do
     @params.permit!
-    @params["person"].each_value { |value| assert(value.permitted?) if value == 32 }
+    @params.each_value do |value|
+      assert_predicate(value, :permitted?)
```

Comment:
Does all each value respond to `:permitted?`?

https://github.com/rails/rails/blob/4de7fdbc37ea002afb68e7c9bf581c34d7cdd566/actionpack/test/controller/parameters/accessors_test.rb#L10-L19